lib/src/formatter.dart

```diff
@@ -14,15 +14,15 @@ import 'util/score_utils.dart';
 // Number of times to perform linting to get stable benchmarks.
 const benchmarkRuns = 10;
 
-String getLineContents(int lineNumber, AnalysisError error) {
+String getLineContents(int? lineNumber, AnalysisError error) {
   final path = error.source.fullName;
   final file = File(path);
   String failureDetails;
   if (!file.existsSync()) {
```

Comment:
It isn't part of the migration, but there's no value in testing whether the file exists. Just read the content and catch the exception if the file doesn't exist.